test/unit/math/prim/functor/ode_test_functors.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef TEST_UNIT_MATH_PRIM_FUNCTOR_ODE_TEST_FUNCTORS_HPP
+#define TEST_UNIT_MATH_PRIM_FUNCTOR_ODE_TEST_FUNCTORS_HPP
+
+#include <stan/math/prim.hpp>
+
```

Comment:
Can we put all of these in `namespace stan { namespace test {`? You can also put them into a [test fixture](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests) so they are available locally to the  ODE stuff. If you do the test fixture route would also be good to add `SetUp()` and `TearDown()` functions to make sure memory is recovered

Reply:
I don't think there's any special allocation here, so if you want to add all the ODE tests to `AgradRev` here: https://github.com/stan-dev/math/pull/2043, that's fine and I'll defer writing anything separate here.

I can put the other things in a namespace.

Reply:
Yeah there's no special allocation, recovering memory after is just to make sure each test get's a reset arena. It's not a biggie but is probably good to have we do the `jumbo` tests.

Reply:
If we wanna wait on #2043 to get in to add `AgradRev` to everything we can, otherwise this should be good to merge (and can just make everything `AgradRev` over there).